tests/test_termgraph.py

```diff
@@ -0,0 +1,71 @@
+from unittest import TestCase
+from termgraph import CandleStickGraph, Candle
+
+
+class CandleTests(TestCase):
```

Comment:
I will personally avoid `unittest.TestCase` but would use `py.test` (not only as a test runner... but also to describe tests).

But it can be achieved in an other PR (that's probably safer)